pronto-reek.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency('pronto', '~> 0.6.0')
-  s.add_dependency('reek', '~> 3.11', '>= 3.7.0')
+  s.add_dependency('reek', '~> 4.1.1', '>= 3.7.0')
```

Comment:
Hi, I'm one of the maintainers of Reek, and Reek conforms to semver since 4.0. I therefore highly recommend depending on something like `~> 4.1`. The `>= 3.7.0` is definitely not necessary, since versions below 4 are already excluded by the first version specification.